modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -1636,9 +1638,18 @@ public void testDeleteObjectDefinition() throws Exception {
 		_objectDefinitionLocalService.updateRootObjectDefinitionId(
 			objectDefinition.getObjectDefinitionId(), 0);
 
+		ClassName classNameEntry = ClassNameLocalServiceUtil.fetchClassName(
+			objectDefinition.getClassName());
```

Comment:
Rename it to `className` instead of `classNameEntry`.
See the type of the entity you are creating to help you name your variable.